src/components/cards/LibraryGameCard.vue

```diff
@@ -24,29 +31,41 @@
       </template>
 
       <template v-slot:metadata>
-        <b-icon-person-fill class="text-muted mr-2"></b-icon-person-fill>
-        <span class=" text-muted">{{
-          num_players(game.game.min_players, game.game.max_players)
-        }}</span>
-        <b-icon-clock-fill
-          class="ml-4 mr-2 text-muted"
-          font-scale="0.8"
-        ></b-icon-clock-fill>
-        <span class=" text-muted"
-          >{{ playtime(game.game.min_playtime, game.game.max_playtime) }}
-        </span>
+        <div v-if="!$store.getters['users/current'].is_staff">
+          <metadata-item
+            icon="person-fill"
+            class="d-inline-block"
+            :text="
+              num_players(
+                game.game.min_players,
+                game.game.max_players,
+              ).toString()
+            "
+          />
 
-        <b-icon-briefcase-fill
-          class="text-muted ml-4 mr-2"
-        ></b-icon-briefcase-fill>
+          <metadata-item
 
-        <span class=" text-muted">{{ game.owner.name }}</span>
+            icon="clock-fill"
+            :text="
+              playtime(
+                game.game.min_playtime,
+                game.game.max_playtime.toString(),
+              )
```

Comment:
Is the `toString` in the right place? (Is it needed?)
```suggestion
              playtime(
                game.game.min_playtime,
                game.game.max_playtime,
              ).toString()
```

Reply:
Yep, it is in the wrong place, I needed it before realizing that props can have multiple types, so I changed metadata-item component and removed toString()